### DISCUSSION TITLE
Sardinian TV channels Italy

### DISCUSSION THREAD

**Discussion by Gianni737** [iptv-org/community#1073] [Q&A]
Hello I am looking for streams of these Sardinian TV channels.

Catalan TV
https://www.catalantv.it/

RTS Radio Televisione Sarda
http://radiotelevisionesarda.tv/

Radio Supersound TV
https://www.radiosupersound.it/

Canale Italia Sardegna

TeleSardegna
https://telesardegna.it

L'Unione TV (Videolina2)

Maybe someone knows an IPTV service that has this in its program? 
Many thanks

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
